inc/sitemaps/class-author-sitemap-provider.php

```diff
@@ -96,7 +96,7 @@ public function get_index_links( $max_entries ) {
 	 */
 	protected function get_users( $arguments = [] ) {
 
-		global $wpdb;
+		global $wpdb, $wp_version;
```

Comment:
This change should be removed as the WP version is retrieved in a different way now.

Reply:
Yeah, just did, a mistake while fixing conflicts. Thanks!